src/App.tsx

```diff
@@ -1,54 +1,87 @@
-import React from 'react';
-import './App.scss';
+import React, {
+  useState, useRef, useMemo, useCallback, useEffect,
+} from 'react';
+import debounce from 'lodash.debounce';
+import { Person } from './types/Person';
 import { peopleFromServer } from './data/people';
 
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+export const App = () => {
+  const [people] = useState<Person[]>(peopleFromServer);
```

Comment:
do you need the state for this variable?
```suggestion
  const people = peopleFromServer;
```